src/utils/array.ts

```diff
@@ -7,10 +7,8 @@ function fillEmptyElemetns(heatmaps: number[][]) {
         }
         fullArray.push([i, j, 0]);
     }
-    for (let counter = 0; counter < fullArray.length; counter++) {
-        if (heatmaps[counter]) {
-            fullArray[((heatmaps[counter][0] * 24) + heatmaps[counter][1]) - 1] = heatmaps[counter]
-        }
+    for (let counter = 0; counter < heatmaps.length; counter++) {
+        fullArray[((heatmaps[counter][0] * 24) + heatmaps[counter][1]) - 1] = heatmaps[counter]
     }
     return fullArray;
 }
```

Comment:
The function doesn't work properly. this is the output I got.

`[[0,1,0],[0,2,0],[0,3,0],[0,4,0],[0,5,0],[0,6,0],[0,7,0],[0,8,0],[0,9,0],[0,10,0],[0,11,0],[0,12,0],[0,13,0],[0,14,0],[0,15,0],[0,16,0],[0,17,0],[0,18,0],[0,19,0],[0,20,0],[0,21,0],[0,22,0],[0,23,0],[0,24,0],[1,1,0],[1,2,0],[1,3,0],[1,4,0],[1,5,0],[1,6,0],[1,7,0],[1,8,0],[1,9,0],[1,10,0],[1,11,0],[1,12,0],[1,13,0],[1,14,0],[1,15,0],[1,16,0],[1,17,0],[1,18,0],[1,19,0],[1,20,0],[1,21,0],[1,22,0],[1,23,0],[1,24,0],[2,1,0],[2,2,0],[2,3,0],[2,4,0],[2,5,0],[2,6,0],[2,7,0],[2,8,0],[2,9,0],[2,10,0],[2,11,0],[2,12,0],[2,13,0],[2,14,0],[2,15,0],[2,16,0],[2,17,0],[2,18,0],[2,19,0],[2,20,0],[2,21,0],[2,22,0],[2,23,0],[2,24,0],[3,1,0],[3,2,0],[3,3,0],[3,4,0],[3,5,0],[3,6,0],[3,7,0],[3,8,0],[3,9,0],[3,10,0],[3,11,0],[3,12,0],[3,13,0],[3,14,0],[3,15,0],[3,16,0],[3,17,0],[3,18,0],[3,19,0],[3,20,0],[3,21,0],[3,22,0],[3,23,0],[3,24,0],[4,1,0],[4,2,0],[4,3,0],[4,4,0],[4,5,0],[4,6,0],[4,7,0],[4,8,0],[4,9,0],[4,10,0],[4,11,0],[4,12,0],[4,13,0],[4,14,0],[4,15,0],[4,16,0],[4,17,0],[4,18,0],[4,19,0],[4,20,0],[4,21,0],[4,22,0],[4,23,0],[4,24,0],[5,1,0],[5,2,0],[5,3,0],[5,4,0],[5,5,0],[5,6,0],[5,7,0],[5,8,0],[5,9,0],[5,10,0],[5,11,0],[5,12,0],[5,13,0],[5,14,0],[5,15,0],[5,16,0],[5,17,0],[5,18,0],[5,19,0],[5,20,0],[5,21,0],[5,22,0],[5,23,0],[5,24,0],[6,1,0],[6,2,0],[6,3,0],[6,4,0],[6,5,0],[6,6,0],[6,7,0],[6,8,0],[6,9,0],[6,10,0],[6,11,0],[6,12,0],[6,13,0],[6,14,0],[6,15,0],[6,16,0],[6,17,0],[6,18,0],[6,19,0],[6,20,0],[6,21,0],[6,22,0],[6,23,0],[6,24,0]]`

for the input below

`[310,83,188,45,218,2,329,309,356,307,166,347,87,363,215,378,211,283,350,183,328,192,330,100,269,51,177,158,393,45,137,12,20,76,232,312,195,98,320,226,197,319,371,207,317,220,108,307,332,372,64,185,28,303,192,92,142,163,172,192,380,182,149,103,92,12,91,227,72,377,187,88,104,233,394,246,241,339,154,279,330,111,218,314,204,24,53,279,310,273,187,296,328,154,374,293,203,208,21,376,218,0,261,264,394,164,24,146,395,102,13,372,166,194,243,265,34,281,391,281,344,30,369,0,311,45,231,365,195,399,369,294,90,56,363,299,180,31,116,297,365,53,241,62,386,38,76,236,335,156,360,281,130,99,180,106,35,263,380,394,194,358,352,313,20,13,176,170]`